src/protagonist/Orchestrator.Tests/Integration/AuthHandlingTests.cs

```diff
@@ -54,6 +54,8 @@ public async Task Get_Clickthrough_UnknownCustomer_Returns400()
         
         // Assert
         response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        response.Headers.CacheControl!.NoCache.Should().BeTrue();
+        response.Headers.CacheControl.NoStore.Should().BeTrue();

```

Comment:
Is it okay to cache a 400 for bad customer, even if it's briefly? Alleviate any load on server if we keep getting repeated requests for a customer that's deleted?

Reply:
I think that might require a custom attribute as the `ResponseCacheAttribute` doesn't appear to support doing that - makes sense to have it though